template/template.go

```diff
@@ -263,6 +263,9 @@ func NewTemplateExpander(
 
 				return floatToTime(v)
 			},
+			"now": func() time.Time {
+				return time.Now()
+			},
```

Comment:
To stay consistent with the other functions, let's return a "standard Prometheus timestamp here, i.e. 

```suggestion
			"now": model.Now,
```

The result can be converted to a Go time with the existing `toTime`.